package.json

```diff
@@ -124,7 +125,7 @@
     "@types/tar-stream": "1.6.0",
     "@types/through2": "2.0.33",
     "@types/which": "^1.3.2",
-    "catch-uncommitted": "^1.3.0",
+    "catch-uncommitted": "^1.4.0",
```

Comment:
This was actually necessary to avoid failing on uncommitted changes to `.versionbot/CHANGELOG.yml`.